features/upgrade/node/hpa/upgrade.feature

```diff
@@ -0,0 +1,100 @@
+Feature: basic verification for upgrade testing
+  # @author weinliu@redhat.com
+  @upgrade-prepare
+  @users=upuser1
+  @admin
+  Scenario: Upgrade - Make sure multiple resources work well after upgrade - prepare
+    Given I switch to the first user
+    # Given I switch to cluster admin pseudo user
+    When I run the :new_project client command with:
+      | project_name | node-upgrade |
+    And I use the "node-upgrade" project
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/features/tierN/testdata/infrastructure/hpa/hpa-v2beta1-rc.yaml |
+    Then the step should succeed
+    Given 1 pods become ready with labels:
+      | run=hello-openshift |
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/features/tierN/testdata/infrastructure/hpa/resource-metrics-cpu.yaml |
+    Then the step should succeed
+    Given I wait up to 300 seconds for the steps to pass:
+    """
+    Then expression should be true> hpa('resource-cpu').min_replicas(cached: false) == 2
+    And expression should be true> hpa.max_replicas == 10
+    And expression should be true> hpa.current_cpu_utilization_percentage == 0
+    And expression should be true> hpa.target_cpu_utilization_percentage == 20
+    And expression should be true> hpa.current_replicas == 2
+    """
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/testdata/daemon/daemonset.yaml |
+    Then the step should succeed
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/testdata/configmap/configmap.yaml |
+      | n | <%= project.name %>                                       |
+    Then the step should succeed	
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/testdata/configmap/pod-configmap-volume1.yaml |
+    Then the step should succeed	
+    And the pod named "dapi-test-pod-1" status becomes :succeeded
+ 
+
+  # @author weinliu@redhat.com
+  # @case_id OCP-13016,OCP-13017,OCP-13019
```

Comment:
We should leave only one test case id there

Reply:
Nice catch!

Reply:
> We should keep only one test case id there

Updated, let me know if it's good to go. Thanks!